arcgis-ios-sdk-samples/Content Display Logic/ContentPList.plist

```diff
@@ -1033,10 +1033,6 @@
 				<string>GraphicsOverlayDictionaryRenderer3D</string>
 				<key>descriptionText</key>
 				<string>Create graphics using a local mil2525d style file and an XML file with key-value pairs for each graphic, and display the military symbols in 3D.</string>
```

Comment:
Now that the sample is using a portal item, this description ("using a local…style file") is no longer correct.

Reply:
I made some slight adjustments to the README and description.